### DISCUSSION TITLE
404 after signing into Timesketch

### DISCUSSION THREAD

**Discussion by Turb0Yoda** [google/timesketch#2826] [Q&A, quick solutions, support] [answered]
Hello,

I've attempted to stand up a timesketch instance on both Ubuntu 20.04 and 22.x. In both cases, I am able to see the sign-in page, but after logging in, I get a white screen with both Firefox and Edge on my Windows desktop, and Firefox on the Linux host. I've attached a screenshot of the consoles of both Firefox(Left) and Edge(Right) after logging in. All the containers seem to start normally. I tried the adjustment as noted in this issue to no avail. I will add that in the timesketch-web container, there is a index.x.js- but the middle generated string does not match.
https://github.com/google/timesketch/issues/2410

![image](https://github.com/google/timesketch/assets/22864176/dbe9595d-6ce8-4c75-9ade-5310c90efe3e)

**Comment by jkppr** [comment]
Hi,
are you using the latest timesketch version from the master branch?
Please share your steps and any guides you are following to set up your instance.

**Reply by Turb0Yoda** [reply]
Hello,
Yes. I was following the steps given on the master page here: https://github.com/google/timesketch/blob/master/docs/guides/admin/install.md

**Comment by jkppr** [comment, marked as answer]
Should be fixed with PR #2828

**Reply by Turb0Yoda** [reply]
Will try pulling again now..

**Reply by Turb0Yoda** [reply]
Good now, thank you!
